specification/schemas/LocalitiesAutocompleteRelatedPostalCode.yml

```diff
@@ -0,0 +1,14 @@
+title: LocalitiesAutocompleteRelatedPostalCode
+type: object
+description: Contains a set of related elements to the locality suggestion.
+properties:
+  postal_codes:
```

Comment:
Not good here.
`postal_codes` should not be present (it's already the parent prop) but only sub-properties.

<img width="713" alt="image" src="https://github.com/Woosmap/openapi-specification/assets/1224805/b56ca49a-2a4d-4b52-aa17-93d62e1481b2">

Reply:
to check the rendered spec : https://developers.woosmap.com/api-reference/?ref=https://raw.githubusercontent.com/Woosmap/openapi-specification/bda34125c156a9c62020c5935cfae077842e91af/dist/merged-woosmap-openapi3.json#tag/Localities-API/operation/localitiesAutocomplete